### ISSUE TITLE
invalid i18n message names in messages.json

### ISSUE THREAD

**Issue by illucent** [passff/passff#223]
chromium linux64,
src/modules/menu.js issue:

```
Failed to load extension from: ~/*/passff-1.0.0/src
Name of a key "passff.button.context.label" is invalid. Only ASCII [a-z], [A-Z], [0-9] and "_" are allowed.


Failed to load extension from: ~/*/passff-1.0.0/src
Name of a key "passff.button.root.label" is invalid. Only ASCII [a-z], [A-Z], [0-9] and "_" are allowed.
```

**Comment by tuxor1337** [comment]
Thanks for your feedback!

It's very helpful since I was not aware that, [in fact](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/API/i18n/Locale-Specific_Message_reference), the message names in `messages.json` are not allowed to contain any characters different from [a-z], [A-Z], [0-9], "@" and "_". Even though Firefox seems to accept our message names, we should eventually replace all dots by underscores.

Why do we currently use dots in our message names? That's how localized strings used to be named in legacy versions of PassFF where localization was achieved using property files and so-called `stringbundles` (gsfm = getStringFromName). Using dots was a common naming scheme inside of property files (cf. [Mozilla's XUL tutorial](https://developer.mozilla.org/en-US/docs/Mozilla/Tech/XUL/Tutorial/Property_Files)).

**Comment by tuxor1337** [comment]
Ah, I see, this has already been addressed in #213 !

**Comment by illucent** [comment]
Thanks!
yes CamelCase seems okay (I've tried to rename 'passff.button.context.label' to 'passffButtonContextLabel')
